src/demo-app/list/list-demo.html

```diff
@@ -31,6 +31,7 @@ <h4 mat-line>{{message.from}}</h4>
           <span>{{message.subject}} -- </span>
           <span class="demo-secondary-text">{{message.message}}</span>
         </p>
+        <mat-divider [inset]="true" *ngIf="!last"></mat-divider>
```

Comment:
You forgot to bind `last` to a local variable: `*ngFor="let message of messages; last as last"`

Reply:
Added, this got cut from a previous commit when rebased, good catch